client/me/billing-history/index.jsx

```diff
@@ -58,7 +58,7 @@ module.exports = React.createClass( {
 									'A complete history of all billing transactions for your WordPress.com account. If you are looking to add or cancel a plan go to {{link}}My Upgrades{{/link}}.',
```

Comment:
Need to update this copy as well:
If you are looking to add or cancel a plan go to {{link}}Manage Purchases{{/link}}.